app/README.md

```diff
@@ -71,15 +71,15 @@ The filter `setid` is used in `/app/.gitattributes` so that it replaces your app
 #### Install the app to your Jira instance
 
 1. Build your app by running `yarn build` in **/app/jenkins-for-jira-ui**.
-2. Deploy your app by running `forge deploy` in **/app**.
+2. Deploy your app by running `forge deploy` in **/app**. `forge deploy --no-verify` if encounter invalidation of package.
```

Comment:
Should we be doing this @joshkay10?

Reply:
yeah we gotsta, since build and deployment info arent "valid" moduels at the moment, 
Maybe we break it into two steps so we are super careful.
initially run
`forge deploy` or `forge lint`
and if everything is good except for 2 acceptable errors...
- invalid value 'devops:deploymentInfoProvider' in modules
- invalid value 'devops:buildInfoProvider' in modules 

Then run with the no-verifuy flag

Then its less of a mystery

Reply:
Yeah this is a hard one. Either make the devloop more complex or risk other errors coming through. Will any of these legitimate errors be caught in a PR build?